__tests__/app.test.js

```diff
@@ -31,8 +32,32 @@ describe("GET /api/topics", () => {
       });
   });
   test("404: should respond with 'Not found' for invalid endpoint", () => {
+    return request(app).get("/api/banana").expect(404);
+  });
+});
+
+const generateEndpointObject = () => {
+  const endpoints = require("../endpoints.json");
+  const expectedResponse = {};
+
+  for (const endpoint in endpoints) {
+    expectedResponse[endpoint] = {
+      ...endpoints[endpoint],
+    };
+  }
+
+  return expectedResponse;
+};
+
+describe("GET /api", () => {
+  test("200 should respond with a JSON object of available endpoints", () => {
+    const expectedResponse = generateEndpointObject();
+
     return request(app)
-      .get("/api/banana")
-      .expect(404)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
we can just expect that the response is the same as the required in endpoint.json